### ISSUE TITLE
Take log level from environment variable / command-line parameter and set the logger's configuration

### ISSUE THREAD

**Issue by kbr-** [scylladb/scylla-cdc-java#15]
The library and replicator use FINE and FINER log levels for certain messages, but it's currently not possible to see those messages since the logger is configured to show only higher level messages.
It should be possible to configure the logger's level when starting the replicator.

**Comment by haaawk** [comment]
It should be possible to change the log level. See http://tutorials.jenkov.com/java-logging/configuration.html

> The Java Logging API has a default logging configuration file located at "lib/logging.properties", inside the JRE directory. If you edit this file, you edit the default logging settings for the entire JRE, for every program executed. This is most often not what you want to do, though.
> Instead, you can set a separate configuration file for your application. You do so by setting the JVM property java.util.logging.config.file to point to this file.
> The configuration file is a standard property file as you know them from Java. Inside this property file you can set properties that configure the various Logger's and Handler's used in your application.

**Comment by haaawk** [comment]
@kbr- please confirm whether there's really a problem or we can just close this issue.

**Comment by kbr-** [comment]
That's a lot of effort to just change the log level.
Anyway, the README should explain how to do it so we don't need to lookup tutorials on the internet.
I did the following:
1. create a `replicator.properties` file with the following contents:
```
handlers = java.util.logging.ConsoleHandler
.level = FINER
java.util.logging.ConsoleHandler.level = FINER
```
2. run replicator using the following command:
```
java -Djava.util.logging.config.file=replicator.properties -cp replicator.jar com.scylladb.cdc.replicator.Main -k ks1 -t table1 -s 127.0.0.10 -d 127.0.0.20 -cl one -m delt
```

I also noticed that additionally to the "Querying window" logs (which I wanted to see):
```
Feb 17, 2021 1:45:52 PM com.scylladb.cdc.cql.driver3.Driver3WorkerCQL query                                                                                   
FINER: Querying window: [d8621930-711d-11eb-8080-808080808080, ea43bc2f-711d-11eb-7f7f-7f7f7f7f7f7f] for task: TaskId(GenerationId(Timestamp(2/17/21 1:27 PM)), VNodeId(247), TableName(ks1, table1)), task state: TaskState(Timestamp(2/17/21 1:44 PM), Timestamp(2/17/21 1:44 PM), Optional.empty)
```
I'm getting a bunch of uninteresting logs like this:
```
Feb 17, 2021 1:45:59 PM com.sun.jmx.mbeanserver.Repository remove                                                                                             
FINER: name = cluster2-metrics:name=trashed-connections                                                                                                                                                                                                                                                                       
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor unregisterMBean                                                                 
FINER: Send delete notification of object cluster2-metrics:name=trashed-connections                                                                                                                                                                                                                                           
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor sendNotification                                                                
FINER: JMX.mbean.unregistered cluster2-metrics:name=trashed-connections                                                                                                                                                                                                                                                       
Feb 17, 2021 1:45:59 PM com.sun.jmx.mbeanserver.Repository retrieve                                                                                           
FINER: name = cluster2-metrics:name=ignores                                                                                                                                                                                                                                                                                   
Feb 17, 2021 1:45:59 PM com.sun.jmx.mbeanserver.Repository remove                                                                                             
FINER: name = cluster2-metrics:name=ignores                                                                                                                                                                                                                                                                                   
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor unregisterMBean                                                                 
FINER: Send delete notification of object cluster2-metrics:name=ignores                                                                                                                                                                                                                                                       
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor sendNotification                                                                
FINER: JMX.mbean.unregistered cluster2-metrics:name=ignores                                                                                                                                                                                                                                                                   
Feb 17, 2021 1:45:59 PM com.sun.jmx.mbeanserver.Repository retrieve                                                                                           
FINER: name = cluster2-metrics:name=inflight-requests                                                                                                                                                                                                                                                                         
Feb 17, 2021 1:45:59 PM com.sun.jmx.mbeanserver.Repository remove                                                                                             
FINER: name = cluster2-metrics:name=inflight-requests                                                                                                                                                                                                                                                                         
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor unregisterMBean                                                                 
FINER: Send delete notification of object cluster2-metrics:name=inflight-requests                                                                                                                                                                                                                                             
Feb 17, 2021 1:45:59 PM com.sun.jmx.interceptor.DefaultMBeanServerInterceptor sendNotification                                                                
FINER: JMX.mbean.unregistered cluster2-metrics:name=inflight-requests                                                                                                                                                                                                                                                         
```

Perhaps it would be better to increase the level of "Querying window" to FINE?

**Comment by haaawk** [comment]
I agree - we should add it to README.

**Comment by avelanarius** [comment]
> Perhaps it would be better to increase the level of "Querying window" to FINE?

Fixed this aspect in d99da3099c9a08bc818dea87914ade5569e80c2e. The other thing (.properties file and better documentation) is nearly finished.
